src/multipliers/PermanentNFTMultiplier.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.22;
+
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import {INFTMultiplier} from "src/interfaces/multipliers/INFTMultiplier.sol";
+
+/// @title Permanent NFT Multiplier
+/// @notice This contract provides a permanent multiplying factor for users based on NFT ownership
+/// @dev Implements the INFTMultiplier interface
+contract PermanentNFTMultiplier is INFTMultiplier {
+    /// @notice The address of the NFT contract
+    IERC721 public immutable NFTAddress;
```

Comment:
immutable variables are formatted the same as constant; SNAKE_CASE

Reply:
addressed with 706cd8e